packages/store-consumer/src/experimental/Context.sol

```diff
@@ -4,7 +4,7 @@ pragma solidity >=0.8.24;
 // @dev Provides information about the current execution context
 // We only use it in these contracts in case we want to extend it in the future
 abstract contract Context {
-  function _msgSender() internal view virtual returns (address) {
+  function _msgSender() public view virtual returns (address) {
```

Comment:
ooc why public for such a small thing?

Reply:
because `_msgSender` in `WorldContext.sol` is public so we need to be able to override

Reply:
for context: it's public to have a unique interface id we can check for in `supportsInterface` so only contracts that are intended to be used as systems can be registered as systems